components/push/src/communications.rs

```diff
@@ -468,7 +468,7 @@ mod test {
             .with_body(body)
             .create();
             let mut conn = connect(config.clone(), None, None).unwrap();
-            let channel_id = hex::encode(crate::crypto::get_bytes(16).unwrap());
+            let channel_id = hex::encode(crate::crypto::get_random_bytes(16).unwrap());
```

Comment:
Sorry, I just, I couldn't handle the name of this function - what *sort* of bytes are we getting??